pandas/core/strings.py

```diff
@@ -680,12 +680,35 @@ def str_repeat(arr, repeats):
 
     Parameters
     ----------
-    repeats : int or array
-        Same value for all (int) or different value per (array)
+    repeats : int or sequence of int
+        Same value for all (int) or different value per (sequence)
 
     Returns
     -------
-    repeated : Series/Index of objects
+    Series or Index of object
+        Series or Index of repeated string object specified by input parameter repeats
```

Comment:
I think this line will be too long (pep8 80 chars)

also "string object" -> "string objects"